canvas_account_admin_tools/views.py

```diff
@@ -55,40 +56,38 @@ def lti_launch(request):
 
 
 @login_required
-@lti_role_required(const.ADMINISTRATOR)
-@lti_permission_required(settings.PERMISSION_ACCOUNT_ADMIN_TOOLS)
+@lti_permission_required('account_admin_tools')
 @require_http_methods(['GET'])
 def dashboard_account(request):
-    custom_canvas_account_sis_id = request.LTI['custom_canvas_account_sis_id']
-    custom_canvas_membership_roles = request.LTI['custom_canvas_membership_roles']
-    conclude_courses = settings.CONCLUDE_COURSES_URL
+    tools = ['conclude_courses',
+             'course_info',
+             'cross_listing',
```

Comment:
Not actually a tool name, but a permission name; perhaps changing this list variable's name, or making a comment that it needs to match the item in `TOOL_PERMISSIONS`, would make this clearer.